src/nbl/asset/utils/CIESProfile.h

```diff
@@ -0,0 +1,137 @@
+// Copyright (C) 2018-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef __NBL_ASSET_C_IES_PROFILE_H_INCLUDED__
+#define __NBL_ASSET_C_IES_PROFILE_H_INCLUDED__
+
+#include "nbl/asset/metadata/CIESProfileMetadata.h"
+#include "nbl/core/Types.h"
+#include <sstream>
+
+namespace nbl 
+{
+    namespace asset 
+    {
+        class CIESProfile 
+        {
+            public:
+                using IES_STORAGE_FORMAT = double;
+
+                _NBL_STATIC_INLINE_CONSTEXPR size_t CDC_DEFAULT_TEXTURE_WIDTH = 1024;
+                _NBL_STATIC_INLINE_CONSTEXPR size_t CDC_DEFAULT_TEXTURE_HEIGHT = 1024;
```

Comment:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/630#discussion_r1431351882
https://github.com/Devsh-Graphics-Programming/Nabla/pull/630#discussion_r1431357799

Reply:
you can work out the Octagonal UV coordinates of the `(theta,phi)` points as you integrate and their Quadrilateral in UV space

Reply:
ok going to a later PR